lightning/src/ln/msgs.rs

```diff
@@ -2080,7 +2080,7 @@ mod fuzzy_internal_msgs {
 	pub struct InboundTrampolineEntrypointPayload {
 		pub amt_to_forward: u64,
 		pub outgoing_cltv_value: u32,
-		pub multipath_trampoline_data: FinalOnionHopData,
+		pub multipath_trampoline_data: Option<FinalOnionHopData>,
```

Comment:
Is this the discussion you're referring to in the commit message? https://github.com/lightning/bolts/pull/836#discussion_r1987831330

Reply:
I'd agree, although it's all cfg-gated for now anyway I suppose... @arik-so was the reason for doing this now for ease of testing?

Reply:
But generating that secret is something to be looked at when implementing the forwarding part isn't it? Isn't it best for now to stick with the proposed spec?